src/project.cc

```diff
@@ -472,6 +472,15 @@ std::vector<Project::Entry> LoadFromDirectoryListing(ProjectConfig* config) {
                      }
                    });
 
+  optional<std::string> maybe_cfg = GetGlobalConfigDirectory();
```

Comment:
Is the purpose of this block only for the LOG_S(INFO)? In that case we can move the LOG_S(...) into LoadFromDirectoryListing which should eliminate the need for this block.

Reply:
No sure what you're referring to here? It that the right block?